src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerToolInstaller.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2015 Jesse Glick, CloudBees Inc.
```

Comment:
IIUC our employer is intending to deliver guidelines on the topic so it need not be discussed here.